ShvetsovAM/lab1/Calculator.code

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <conio.h>
+
+using namespace std;
+
+int main()
+{
+	setlocale(0, "");
+	float first, second, result;
+	char sign;
+	cout << "Введите первое число ";
+	cin >> first;
+	cout << "Введите второе число ";
+	cin >> second;
+	cout << "Введите знак ";
+	cin >> sign;
+	switch (sign)
+	{
+	case '+': cout << first << " " << sign << " " << second << " = " << first + second;
```

Comment:
You have to store the result in variable and print it after all checks

Reply:
I dont know how to do this
It's not working
But how i did - working